rspace/src/main/scala/coop/rchain/rspace/HotStore.scala

```diff
@@ -251,6 +253,14 @@ private class InMemHotStore[F[_]: Sync, C, P, A, K](
         )
       )
 
+  def toMap: F[Map[Seq[C], Row[P, A, K]]] =
+    for {
+      cache         <- S.read
+      data          = cache.data.readOnlySnapshot().map { case (k, v) => (Seq(k), v) }.toMap
+      continuations = (cache.continuations ++ cache.installedContinuations.mapValues(Seq(_))).toMap
+      merged        = merge(data, continuations, Seq.empty[Datum[A]], Seq.empty[WaitingContinuation[P, K]])
+      mapped        = merged.mapValues { case (d, k) => Row(d, k) }
```

Comment:
And this is a usecase for Function.tupled, which I prefer instead of using the partial function:
```mapped = merged.mapValues(Function.tupled(Row(_,_)))```
...but, if you don't like it, I don't insist

Reply:
How is this preventing an approval?